### PR TITLE
Update gcs version

### PR DESCRIPTION
**Pull request by elonazoulay**
Update google cloud jars to latest version.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1217,6 +1217,7 @@
                                 <excludes combine.children="append">
                                     <!-- TODO: fix this in Airlift resolver -->
                                     <exclude>org.codehaus.plexus:plexus-utils</exclude>
+                                    <exclude>com.google.guava:guava</exclude>
```
Comment (findepi) [inline pom.xml]: Just curious, what's this "TODO: fix this in Airlift resolver" is about?

Comment (elonazoulay) [reply pom.xml]: Not sure, that was there since November. @electrum, would you know about this?

Comment (findepi) [reply pom.xml]: Oh, I misunderstood. Maybe we don't need to add guava to excludes here?

Comment (electrum) [reply pom.xml]: That comment is specifically for `plexus-utils`. I don’t remember the details but should be fairly obvious if you remove it.

We do need the exclude for Guava.

**Comment by electrum** [comment]
For release notes, something like this?

> Hive Connector
> Improve performance when reading data from GCS.

**Comment by elonazoulay** [comment]
Sounds good! Will wait until it's accepted.

**Pull request merged**